### DISCUSSION TITLE
Can't it simulate random card dealing games like RLCard?

### DISCUSSION THREAD

**Discussion by XliOK** [Cryolite/kanachan#63] [Q&A]
Can't it simulate random card dealing games like RLCard?

**Comment by Cryolite** [comment]
What kind of simulation are you inquiring about? Is it for evaluation purposes, or is it for reinforcement learning? The former has been already implemented, while the latter is still in the process of being implemented.

**Reply by XliOK** [reply]
It is not necessary to prepare a dataset, but rather to generate a dataset through playing Mahjong with an agent and then continue training by playing Mahjong repeatedly

**Reply by XliOK** [reply]
Can anyone help answer my question?
